src/types/PaymentSheet.ts

```diff
@@ -37,11 +37,13 @@ export declare namespace PaymentSheet {
     | {
         googlePay?: true;
         merchantCountryCode: string;
+        currencyCode?: string;
```

Comment:
```suggestion
        currencyCode: string;
```
if `googlePay === true`, then the currencycode _must_ be supplied, right?

Reply:
Only when used with a setup intent for "setup mode". When used for _payment_ (with a payment intent), the currency is inherited from the payment intent (where it is a required parameter). 

The `SetupParams` type name I think is a bit of a misnomer here, or overlaps with Stripe nomenclature with Setup Intents, but I believe refers generally to _setting up the payment sheet_. The `ClientSecretParams ` for example provide cases to handle both payment intent and setup intent usage.